docker-compose-openmrs.yml

```diff
@@ -16,13 +16,14 @@ services:
       OMRS_CONFIG_CONNECTION_USERNAME: ${OPENMRS_DB_USER:-openmrs}
       OMRS_CONFIG_CONNECTION_PASSWORD: ${OPENMRS_DB_PASSWORD:-openmrs}
       HOST_URL: https://${O3_HOSTNAME}
+      OAUTH2_ENABLED: ${ENABLE_SSO}
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://localhost:8080/openmrs/health/started" ]
       interval: 10s
       timeout: 5s
       retries: 48
       start_period: 120s
-    image: openmrs/openmrs-reference-application-3-backend:3.1.1
+    image: openmrs/openmrs-core:2.6.7
```

Comment:
Should we use `O3_DOCKER_IMAGE_TAG` instead of hard coding?

Reply:
`O3_DOCKER_IMAGE_TAG` was common to both the backend and frontend, so it made sense to extract it as an env var so we could change it in one place. Now that we are using different Docker images, it is fine to treat it like other apps, e.g., Odoo, and SENAITE docker-compose services.